backend/api/Services/MapService.cs

```diff
@@ -154,14 +154,14 @@ private Boundary ExtractMapMetadata(BlobItem map)
                     double.Parse(map.Metadata["upperRightY"], CultureInfo.CurrentCulture) / 1000;
                 return new Boundary(lowerLeftX, lowerLeftY, upperRightX, upperRightY);
             }
-            catch (FormatException e)
+            catch (Exception e) when (e is FormatException || e is KeyNotFoundException)
```

Comment:
We should probably catch all errors here to log the warning and reraise
```suggestion
            catch (Exception e)
```